app/views/past_chancellors/index.html.erb

```diff
@@ -10,12 +10,12 @@
   </div>
 </div>
 
-<%= render partial: "century", locals: { heading: "21st century", people: @twentyfirst_century_chancellors } %>
```

Comment:
Sorry I missed this one! It hasn't been deleted, but moved to shared and renamed '_historical_figures_by_century' since the logic is common now between past chancellors and past foreign secretaries.